wurst/_handles/Framehandle.wurst

```diff
@@ -201,46 +263,54 @@ public function framehandle.setValue(real value)
 public function framehandle.minMax(real minValue, real maxValue)
     BlzFrameSetMinMaxValue(this,minValue,maxValue)
 
-/** TODO */
+/** Sets the step size of the frame (e.g. for sliders) */
 public function framehandle.setStepSize(real stepSize)
     BlzFrameSetStepSize(this, stepSize)
 
-/** Sets the width and height of a given frame  */
+/** Sets the width and height of the frame */
 public function framehandle.setSize(real width, real height)
     BlzFrameSetSize(this, width, height)
 
-/** Sets the vertex color of a given frame  */
+/** Sets the vertex color of the model frame */
 public function framehandle.setVertexColor(integer color)
     BlzFrameSetVertexColor(this, color)
 
+/** Unknown */
 public function framehandle.setLevel(integer level)
     BlzFrameSetLevel(this, level)
 
-/** Searches for the frames parent frame, should be used with caution as getting the parent of any top most frame will lead into a void crash */
+/** Searches for the frames parent frame. 
+    Should be used with caution as getting the parent of any top most frame will lead into a void crash */
 public function framehandle.getParent() returns framehandle
     return BlzFrameGetParent(this)
 
-/** Sets the parent frame for given custom created frame */
+/** Sets the parent frame of the frame */
 public function framehandle.setParent(framehandle parent)
     BlzFrameSetParent(this, parent)
 
-/** Obtains the qualified size of height of a given frame (0-1) */
-public function framehandle.height() returns real
+@deprecated("use getHeight() instead") public function framehandle.height() returns real
```

Comment:
As discussed, I think `width` is fine, if no `setWidth` exists.
I guess it exists implicitly via `setSize` - how about adding a setter just for width then?

Reply:
Good idea but for your information, setSize() can't allow for a perfect setWidth() because it forces us to set a certain height while the fdf allow us to define the width without any height. I will still add it.